__tests__/index.tsx

```diff
@@ -148,6 +148,14 @@ describe('VideoRenderer', () => {
     xit('should return the buffered value', () => {
       // TODO: fake multiple ranges
     });
+
+    it('should return error status when the video is errored', () => {
+      const {component, children} = setup();
+
+      component.find('video').simulate('error');
```

Comment:
@iloire so doing something like this with `play` didn't fired the onPlay handler right?

Reply:
This test is passing:

```
    it('should play the video when play action is called', () => {
      const {component} = setup({children});
      component.find('video').simulate('play');
      expect(component.state('status')).toEqual('playing');
    });
```

But this is not (testing the play action is what we want to do):

```
    it('should play the video when play action is called', () => {
      const {component} = setup({children});
      videoActions.play();
      expect(component.state('status')).toEqual('playing');
    });
```

Any ideas?

Reply:
Oh yeah, you are 💯right.
`videoActions.play()` is what we want to test for sure, and I don't think there is any easy way without manually firing that fake event from the element...